api/src/main/java/io/opentelemetry/metrics/DoubleSumObserver.java

```diff
@@ -67,15 +76,12 @@
     @Override
     Builder setConstantLabels(Map<String, String> constantLabels);
 
-    @Override
-    Builder setMonotonic(boolean monotonic);
-
     @Override
     DoubleSumObserver build();
   }
 
   /** The result for the {@link AsynchronousInstrument.Callback}. */
-  interface ResultDoubleObserver {
+  interface ResultDoubleSumObserver {
```

Comment:
Something to think about: this interface is exactly the same as the one in the up-down version. We could think about sharing the interface, calling it something like `DoubleSumResult` and reducing a wee bit of duplicated code.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/1230